doc/articles/using-uno-ui.md

```diff
@@ -573,5 +573,9 @@ As of iOS 9, the system does not allow iPad applications to dictate their orient
 <true/>
 ```
 
+#### Controlling Keyboard on iOS
+
+If a view needs to keep the keyboard open when you tap on it, you can use SetNeedsKeyboard of the Widow.iOS class.
```

Comment:
```suggestion
If a view needs to keep the keyboard opened when tapping on it, use the `Uno.UI.Controls.Window.SetNeedsKeyboard` method.
```

Reply:
Isn't it an attached property? We could give a XAML snippet instead?